spec/models/concerns/aliquotable_spec.rb

```diff
@@ -121,6 +125,18 @@
       end
     end
 
+    context 'when primary aliquot volume for pools is less than used volume' do
+      it 'adds an error' do
+        skip unless Flipper.enabled?(:y24_153__enable_volume_check_pacbio_pool_on_update)
```

Comment:
I dont think this skip is needed anymore as you have set the flag to be always on at the top of the file